packages/tocco-util/src/notifier/notificationActionFactory.js

```diff
@@ -1,25 +1,35 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {actions as toastrActions} from 'react-redux-toastr'
-import _isString from 'lodash/isString'
 import uuid from 'uuid/v4'
-import {Icon} from 'tocco-ui'
+import {Icon, IconTocco} from 'tocco-ui'
 
 import {modalComponent} from './modules/actions'
 import ModalButtons from './modules/modalComponents/ModalButtons'
 import TitleMessage from './components/TitleMessage'
 
-const NotificationIcon = icon => <Icon icon={ _isString(icon) ? icon : 'exclamation-triangle'} size="3x" />
+const typeIconMap = {
+  error: 'times',
+  info: 'info',
+  success: 'check',
+  warning: 'exclamation'
+}
 
 export function getInfoAction(type, title, message, icon, timeOut) {
+  type = Object.keys(typeIconMap).includes(type) ? type : 'info'
+
   const options = {
-    timeOut: timeOut,
-    showCloseButton: true,
-    component: () => <TitleMessage title={title} message={message}/>
+    attention: false,
+    component: () => <TitleMessage title={title} message={message}/>,
+    icon: <Icon icon={icon || typeIconMap[type] || 'info'} size="3x" />,
+    preventDuplicates: true,
+    removeOnHover: type !== 'warning' && type !== 'error' && timeOut > 0,
```

Comment:
cant you move this "is warning or error in a var would make the code more readable